tests/functional/s/string/string_formatting_error.py

```
@@ -1,5 +1,5 @@
 """test string format error"""
-# pylint: disable=print-statement,unsupported-binary-operation,line-too-long
+# pylint: disable=print-statement,unsupported-binary-operation,line-too-long, consider-using-f-string
```

Pylint (3.3.7)

[refactor] Useless option value for 'disable', 'print-statement' was removed from pylint, see https://github.com/pylint-dev/pylint/pull/4942.

(R0022)